gradle.properties

```diff
@@ -16,10 +16,14 @@
 # under the License.
 group=org.apache.samza
 version=1.7.0-SNAPSHOT
+# These 2 are ones that you can override using properties, like:
+# -PscalaSuffix=2.12 -PyarnVersion=2.10.1
 scalaSuffix=2.12
+yarnVersion=2.10.1
+yarn3Version=3.3.4
 
 # after changing this value, run `$ ./gradlew wrapper` and commit the resulting changed files
-gradleVersion=5.2.1
+gradleVersion=6.9.2
```

Comment:
Is this strictly needed for JDK11 runtime support? @mynameborat can you comment on that?

Reply:
No it is not strictly needed to support 11. But everything seems to build and run as expected, so that it was worth at least moving forward to be more up to date. :shrug: I'm ok reverting if needed, but things seem good here.